lexos/templates/scrub.html

```diff
@@ -34,6 +34,7 @@
                 }
                                 });
            tagDict=JSON.stringify(tagDict);
+           console.log(tagDict);
```

Comment:
Mostly Sublime and occasionally Visual Studio Code. I just installed he eslint plugins, but I haven't explored front-end debuggers.